config/prow/cluster/monitoring/grafana_deployment.yaml

```diff
@@ -17,6 +19,8 @@ spec:
       app: grafana
   template:
     metadata:
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: 'runtime/default'
```

Comment:
It looks like this is actually available in securityContext now and the annotations are getting removed in 1.25. Let's try the securityContext version: https://kubernetes.io/docs/tutorials/clusters/seccomp/#create-a-pod-with-a-seccomp-profile-for-syscall-auditing